### DISCUSSION TITLE
Unable to connect unless host firewall is fully disabled

### DISCUSSION THREAD

**Discussion by patrikarvidsson** [qdm12/gluetun#1307] [Q&A]
Hey!

I have set up a gluetun instance that uses wireguard as the VPN protocol and Mullvad as the provider. It functions correctly as long as I allow all ports in my host's firewall, but for security reasons, I do not want to leave all ports open. I am having difficulty determining which port is required for the gluetun instance to connect to the VPN provider. 

I have attempted to open only the ports listed in the debug logs, without any success. Any ideas what could be wrong?

**Comment by qdm12** [comment]
I personally do this and it works fine (with iptables).
Mine is for example (for both openvpn udp and wireguard):

```
iptables -A DOCKER-USER -s $GLUETUN -p udp --dport 1194 -j ACCEPT
iptables -A DOCKER-USER -s $GLUETUN -p udp --dport 51820 -j ACCEPT
iptables -A DOCKER-USER -s $GLUETUN -j DROP
```

where `GLUETUN` is the gluetun container ip

**Reply by iamsarvagyaa** [reply]
not working for me. I did this.
